SnoopCompileCore/src/SnoopCompileCore.jl

```diff
@@ -11,6 +11,9 @@ if VERSION >= v"1.2.0-DEV.573"
     include("snoopi.jl")
 end
 
+if VERSION >= v"1.6.0-DEV.1190"  # https://github.com/JuliaLang/julia/pull/37749
```

Comment:
I swear I commented about this somewhere but I can't find it; it's fine with me if you want to load the parceling even on earlier Julia versions (your choice).

Reply:
Ah, that was on the other PR. I don't think that will work on this PR, because the parcelling depends on the changes in julia. 😁